crates/ruff_python_parser/resources/invalid/statements/match/invalid_mapping_pattern.py

```diff
@@ -0,0 +1,20 @@
+# Starred expression is not allowed as a mapping pattern key
+match subject:
+    case {*key}:
+        pass
+    case {*key: 1}:
+        pass
+    case {*key 1}:
+        pass
+    case {*key, None: 1}:
+        pass
+
+# Pattern cannot follow a double star pattern
+# Multiple double star patterns are not allowed
+match subject:
+    case {**rest, None: 1}:
+        pass
+    case {**rest1, **rest2, None: 1}:
+        pass
+    case {**rest1, None: 1, **rest2}:
+        pass
```

Comment:
Nit: should this have a newline at the end?

Reply:
I don't think it matters but I'll add it anyways. Not sure why my editor removed it lol. Thanks for noticing!